File: mycollab-web/src/main/java/com/esofthead/mycollab/module/crm/events/DocumentEvent.java
package com.esofthead.mycollab.module.crm.events;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;
import com.esofthead.mycollab.module.file.domain.criteria.FileSearchCriteria;

public class DocumentEvent {
	public static class GotoDashboard extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public GotoDashboard(Object source, Object data) {
			super(source, data);
		}
	}

	public static class Search extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public Search(Object source, FileSearchCriteria data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/scala/com/esofthead/mycollab/module/crm/events/DocumentEvent.scala
package com.esofthead.mycollab.module.crm.events

import com.esofthead.mycollab.eventmanager.ApplicationEvent

/**
 * @author MyCollab Ltd.
 * @since 5.0.3
 */
object DocumentEvent {

  class GotoDashboard(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
}
